eng/common/templates/job/job.yml

```diff
@@ -120,6 +120,9 @@ jobs:
         continueOnError: ${{ parameters.continueOnError }}
         condition: and(succeeded(), in(variables['_SignType'], 'real', 'test'), eq(variables['Agent.Os'], 'Windows_NT'))
 
+  - task: CodeQL3000Init@0
+    displayName: CodeQL Initialize
+
```

Comment:
Why is this task called globally? Also, such changes would need to be made in Arcade.

Reply:
I wasn't sure where best to manually inject CodeQL tasks, so this was just where I thought the autoinjection had been placing the tasks in the official pipeline. I just put it here to get things working, but its subject to improvement!
I'm not very familiar, why would the change need to be made in Arcade? I'm still learning how to use `.yml` better, but I'm wondering if theres an easy way to just wrap the job like 
```
      #
      # Build CoreCLR runtime packs
      #
      - template: /eng/pipelines/common/platform-matrix.yml
        parameters:
          jobTemplate: /eng/pipelines/common/global-build-job.yml
          buildConfig: release
          platforms:
          - Linux_x64
          - windows_x64
          jobParameters:
            buildArgs: -s clr+libs+host+packs -c $(_BuildConfig)
            nameSuffix: AllSubsets_CoreCLR
            isOfficialBuild: ${{ variables.isOfficialBuild }}
            timeoutInMinutes: 360
```
within a `CodeQL3000Init@0` and `CodeQL3000Finalize@0` task?

Reply:
> I'm not very familiar, why would the change need to be made in Arcade?

Any change to `eng/common/*` needs to be made in Arcade as those are shared files across all core stack repositories.

Reply:
> Instead of manually injecting the Tasks in eng/common/templates/job/job.yml, I moved them to be just around Build Product in eng/pipelines/common/global-build-job.yml as exemplified in this build

What I'm asking is mostly to add a condition to the task so that it only runs when needed, i.e. via a property.